base/gmp.jl

```diff
@@ -21,10 +21,10 @@ else
 end
 const CdoubleMax = Union{Float16, Float32, Float64}
 
-gmp_version() = VersionNumber(unsafe_string(unsafe_load(cglobal((:__gmp_version, :libgmp), Ptr{Cchar}))))
+version() = VersionNumber(unsafe_string(unsafe_load(cglobal((:__gmp_version, :libgmp), Ptr{Cchar}))))
 gmp_bits_per_limb() = Int(unsafe_load(cglobal((:__gmp_bits_per_limb, :libgmp), Cint)))
 
-const GMP_VERSION = gmp_version()
+const VERSION = version()
 const GMP_BITS_PER_LIMB = gmp_bits_per_limb()
```

Comment:
seems like the `GMP_` and `gmp_` prefix are also a bit redundant with the module name here